### PR TITLE
Feature/improve wam installation documentation

### PR DESCRIPTION
**Pull request by 4lm**
I extended the WAM installation manual to take Windows user into account, because of reference to the WAM docs in the stemp_abw docs for the installation part and there I also included a part for Windows users. Please review this pull request - thx!

Edit: https://github.com/rl-institut/WAM_APP_stemp_abw/blob/feature/fixtures-install/doc/installation_and_setup.rst

### CODE REVIEW

File: doc/getting_started.rst [hunk 1]
```diff
@@ -235,6 +325,71 @@ then add the following two lines
...
+
+- Example app: `stemp_abw <https://github.com/rl-institut/WAM_APP_stemp_abw>`_
+
+.. code:: bash
+
+    conda install -c conda-forge gdal
+    conda activate django
+    conda install shapely
+    pip install -r requirements.txt
+    pip install -r stemp_abw/requirements.txt
```
Comment (Bachibouzouk) [inline doc/getting_started.rst:374]: one can do that with `python install_requirements.py`, it is more general

**Review by Bachibouzouk — COMMENTED** [review]
I unfortunately can't test the installation steps as I only have linux :(. As it is better to have some help than none, I will nevertheless approve the PR.

Apart from that detail, thanks for the improvement of the documentation!

**Comment by nesnoj** [comment]
Thx @4lm for torturing yourself with this...uuuh...what's that called? Windoze?

If you don't mind @Bachibouzouk @henhuy I'd hotfix this into the master (without a release) as this doc is needed for StEmp v1.0 which I'm going to release tomorrow.

**Comment by henhuy** [comment]
don't mind

**Review by henhuy — APPROVED** [review]

**Review by nesnoj — APPROVED** [review]

**Comment by nesnoj** [comment]
I amended the CHANGELOG

**Pull request merged**